python/lsst/obs/base/defineVisits.py

```diff
@@ -573,7 +561,7 @@ def run(
         # This is the only parallel step, but it _should_ be the most expensive
         # one (unless DB operations are slow).
         self.log.info("Computing regions and other metadata for %d visit(s).", len(definitions))
-        allRecords = mapFunc(self._buildVisitRecordsSingle, zip(definitions, itertools.repeat(collections)))
+        allRecords = map(self._buildVisitRecordsSingle, zip(definitions, itertools.repeat(collections)))
```

Comment:
```python
allRecords = [self._buildVisitRecordsSingle(def, collections) for def in definitions]
```
? Seems like we no longer need to be using `itertools.repeat()` there.

Reply:
This also let me remove `_buildVisitRecordsSingle`, which only existed to unpack the `tuple` returned by `zip` into a pair of arguments to `_buildVisitRecords`.